src/app/Components/adminpanel/user-management/user-management.component.ts

```diff
@@ -38,46 +41,63 @@ export class UserManagementComponent implements OnInit {
 
   ngOnInit() {
     this._userListService.getUserList().subscribe((result: any) => {
+      result.users.sort((x1, x2) => x1.id - x2.id);
       this.dataSource = this._userListService.StatusConverter(result.users);
       this.usersAmount = result.count;
       this.isLoadingResults = false;
+      if (this.usersAmount === 0) {
+        this._notificationService.error('There is no users for this request');
+      }
+    },
+      error => {
+        this._notificationService.error(error);
+      });
+
+    this._userListService.getStatuses().subscribe((result: any) => {
+      this.statuses = result;
+      console.log(this.statuses);
```

Comment:
remove console.log

Reply:
ok, I'll check it again,  but as i remember there where some bugs with it :(

Reply:
sorry i can't because mat-pagination really shitty dude if we talking about float